app/database/operator/server_data_operator/comparators/scheduled_post.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import type ScheduledPostModel from '@typings/database/models/servers/scheduled_post';
+
+export function shouldUpdateScheduledPostRecord(e: ScheduledPostModel, n: ScheduledPost) {
+    return Boolean(n.update_at > e.updateAt);
```

Comment:
If we add the `teamID` as we spoke, you may have to add it here.

Reply:
We have not added the extra column for the `teamID` in the schedule post table. This will not be required.